leipzig-gloss.typ

```diff
@@ -98,7 +175,20 @@
         none
     }
 
-    [#gloss_number #pad(left: 1em)[#gloss_items]]
+    style(styles => {
+        block(breakable: breakable)[
+            #stack(
+            dir:ltr,
```

Comment:
Mostly a note, this library needs to be able to support RTL text correctly to be useful to linguists working with Hebrew/Arabic/Farsi/etc.

Reply:
I'm not super familiar with how glosses for RTL languages are formatted (the only such glosses I've personally encountered used Latin transliterations) but I can look into detecting if the surrounding text is RTL and editing things based on that. I think even as-is it should still work fine with RTL text inside the gloss (though requiring the actual lists of words for each line to list the words in LTR order) but I haven't tested that.

Reply:
I think it's fine to not try to solve RTL text in this patch, it's just something I want to keep in mind.